apps/dashboard/src/components/common/SideBar.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useRef, useCallback } from "react";
+import { Moon, Sidebar, Sun, User } from "react-feather";
+import {
+  Button,
+  ButtonSize,
+  ButtonType,
+  Switch,
+  Theme,
+  removeToken,
+  useFormatMessage,
+  useTheme,
+} from "@litespace/luna";
+import { useClosableRef } from "@litespace/luna";
+import cn from "classnames";
+import { useNavigate } from "react-router-dom";
+import { AnimatePresence, motion } from "framer-motion";
+import { useAppDispatch, useAppSelector } from "@/redux/store";
+import { profileSelectors, resetUserProfile } from "@/redux/user/profile";
+import { Route } from "@/lib/route";
+import { SideBarAccordion } from "@/components/common/SideBarAccordion";
+import { NavAccordionItem, NavOption } from "@/types/navbar";
+import SideBarItem from "@/components/common/SideBarItem";
+
+const framerSidebarBackground = {
+  initial: { x: "30rem", y: "0rem" },
+  animate: { x: "0rem", y: "0rem" },
+  exit: { x: "20rem", y: "0rem" },
+  transition: { duration: 0.2 },
+};
```

Comment:
It is a good practice to always assign types to your variables. 
```suggestion
} as const;
```